### PR TITLE
Adding firebase hosting

### PR DESCRIPTION
**Pull request by Estaer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+#firebase
+.firebase
```
Comment (jokamjohn) [inline .gitignore:3]: Why are you ignoring this file?

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "styled-components": "^5.3.0",
-    "web-vitals": "^1.0.1"
+    "web-vitals": "^1.0.1",
+    "yarn": "^1.22.18"
```
Comment (jokamjohn) [inline package.json]: Why have yarn in the dependencies?

File: package.json [hunk 3]
```diff
@@ -37,5 +38,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "firebase-tools": "^10.6.0"
```
Comment (jokamjohn) [inline package.json]: WHy have this in your dependencies, shouldnt this be installed locally than in the repo deps?

File: .gitignore (hunk 1 above)
Comment (Estaer) [reply .gitignore:3]: Since it contains cache files that are created during deployment, didn't find it necessary to add and push

**Review by jokamjohn — APPROVED** [review]

**Pull request merged**